### ISSUE TITLE
Unified Log Parser Fails on Windows

### ISSUE THREAD

**Issue by stark4n6** [mandiant/macos-UnifiedLogs#25]
I'm having issues running on Windows 11, not sure if the command is correct or not.

![image](https://github.com/user-attachments/assets/e4f91c62-fe7e-492b-8d76-8ecbfb8c8fbb)

Logs pulled right from the device via Sysdiagnose. Any thoughts?

**Comment by puffyCid** [comment]
What happens if u specify the output filename?
Like:
```
unifiedlog_parser.exe -i <path> -o "C:\Users\kevin\Desktop\KP 13\out.csv"
```
Is KP 13 a directory?

**Comment by stark4n6** [comment]
@puffyCid yes it's a directly, do you have strictly state a file name and extension as output or can it be a folder? It wasn't very clear by the help menu

**Comment by puffyCid** [comment]
Yes an output filename is required. There have quite commits I'll try to upload a new release and update the help menu

**Comment by stark4n6** [comment]
@puffyCid great, I'll try that, and I'll keep an eye out for a new release (was wondering if a new wrapped exe would be coming), thanks!

**Comment by stark4n6** [comment]
@puffyCid finally got around to this, ran fine with a few errors and warnings, just now have to find something open a 3gb CSV file.

**Comment by stark4n6** [comment]
This seems to be working fine now, not sure if there was any update on a new release. I guess I can close this out

**Comment by puffyCid** [comment]
> This seems to be working fine now, not sure if there was any update on a new release. I guess I can close this out

hey @stark4n6 I just pushed new example binaries on the release page. If you want to though those.
